management/ia/notifications_test.go

```diff
@@ -0,0 +1,242 @@
+package ia
+
+import (
+	"testing"
+
+	"github.com/brianvoe/gofakeit"
+	channelsClient "github.com/percona/pmm/api/managementpb/ia/json/client"
+	"github.com/percona/pmm/api/managementpb/ia/json/client/channels"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/codes"
+
+	pmmapitests "github.com/Percona-Lab/pmm-api-tests"
+)
+
+func TestAddChannel(t *testing.T) {
+	if !pmmapitests.RunIATests {
+		t.Skip("Skipping IA tests until IA will out of beta: https://jira.percona.com/browse/PMM-7001")
+	}
+
+	client := channelsClient.Default.Channels
+
+	t.Run("normal", func(t *testing.T) {
+		_, err := client.AddChannel(&channels.AddChannelParams{
```

Comment:
since we are returning ID now we should check this value whether it is empty or not

Reply:
Done